src/Symfony/Bundle/FrameworkBundle/Command/FormDebugCommand.php

```diff
@@ -0,0 +1,233 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\TableSeparator;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Form\FormRegistryInterface;
+use Symfony\Component\Form\ResolvedFormTypeInterface;
+use Symfony\Component\OptionsResolver\Exception\UndefinedOptionsException;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * A console command for retrieving information about form types.
+ *
+ * @author Yonel Ceruto <yonelceruto@gmail.com>
+ */
+class FormDebugCommand extends Command
+{
+    /** @var OptionsResolver */
+    private $optionsResolver;
+    private $parents = array();
+    private $extensions = array();
+    private $formRegistry;
+
+    public function __construct(FormRegistryInterface $formRegistry)
+    {
+        parent::__construct();
+
+        $this->formRegistry = $formRegistry;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('debug:form')
+            ->setDefinition(array(
+                new InputArgument('class', InputArgument::REQUIRED, 'The form type class'),
+                new InputOption('built-in', null, InputOption::VALUE_NONE, 'Used to detect built-in form types'),
```

Comment:
Do we really need this option?
I'd expect the command to search in chained namespaces `Symfony\\Component\\Form\\Extension\\Core\\Type` and `Symfony\\Bridge\\Doctrine\\Form\\Type` automatically if the provided `class` argument isn't a FQCN.